src/pal/src/cruntime/printfcpp.cpp

```diff
@@ -1145,6 +1146,8 @@ int CoreVfwprintf(CPalThread *pthrCurrent, PAL_FILE *stream, const wchar_16 *for
     PERF_ENTRY(vfwprintf);
     ENTRY("vfwprintf (stream=%p, format=%p (%S))\n",
           stream, format, format);
+    
+    va_copy(ap, aparg);
```

Comment:
Without I broke the AMD64 build, on ARM yes, the copies are not required, but on AMD64 they are.